[*] Running ...
[*] Dataset loaded: /home/lars/workspace/COP/reevo_2/problems/mkp_aco/dataset/train100_dataset.npz with 5 instances.
/home/lars/workspace/COP/reevo_2/problems/mkp_aco/gpt.py:9: RuntimeWarning: invalid value encountered in log
  log_scaled_ratio = np.log(normalized_ratio + 1)
[*] Instance 0: 8.671224027529977
[*] Instance 1: 15.61416521040918
[*] Instance 2: 14.801221644145183
[*] Instance 3: 7.976895700205405
[*] Instance 4: 36.205993893015346
[*] Average:
16.65390009506102
